modules/developer-preview/pages/collections/manage-collections-with-cli.adoc

```diff
@@ -0,0 +1,378 @@
+= Collections via CLI
+
+[abstract]
+_Collections_ can be managed by means of the Couchbase CLI.
+
+[#Using the CLI]
+== Using the CLI
+
+The following sequence of CLI commands demonstrates how to create, examine, and delete scopes and collections.
+It assumes that Couchbase Server has been installed, initialized, and provisioned on a _Ubuntu_ system.
```

Comment:
I think it should be "an Ubuntu system", as the suggested pronunciation of Ubuntu is [ooboontoo](https://ubuntu.com/about). I'll raise a suggestion to add it to the style guide. Not a blocker.